MathTrainerApp/TrainViewController.swift

```diff
@@ -8,32 +8,168 @@
+        case .multiply:
+            return firstNumber * secondNumber
+        default:
+            return 0
+        }
+    }
+    
+    private var answerDivide: Double {
+        print("\(firstNumberDivide)    \(secondNumberDivide)   \(firstNumberDivide / secondNumberDivide)       \((firstNumberDivide / secondNumberDivide * 10).rounded() / 10)")
+        return (firstNumberDivide/secondNumberDivide * 10).rounded() / 10
+    }
+    
     // MARK: - Life cycle
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        configureQuestion()
         configureButtons()
     }
     
+    // MARK: IBActions
+    @IBAction func leftAction(_ sender: UIButton) {
+        check(answer: sender.titleLabel?.text ?? "", for: sender)
+    }
+    
+    @IBAction func rightAction(_ sender: UIButton) {
+        check(answer: sender.titleLabel?.text ?? "", for: sender)
+    }
+    
+    @IBAction func sendData(_ sender: Any) {
+        if (delegate != nil) {
+            let countString: String = String(count)
+            delegate!.sendCount(count: countString, type: type)
+            self.navigationController?.popViewController(animated: true)
+        }
+    }
+    
+    
     //MARK: Methods
     private func configureButtons() {
+        let buttonsAnswer = [leftButton,rightButton]
+        
+        //color
+        buttonsAnswer.forEach { button in
+            button?.backgroundColor = .systemYellow
+        }
         // add shadow
         buttonsAnswer.forEach { button in
-            button.layer.shadowColor = UIColor.darkGray.cgColor
-            button.layer.shadowOffset = CGSize(width: 0, height: 2)
-            button.layer.shadowOpacity = 0.4
-            button.layer.shadowRadius = 3
+            button?.layer.shadowColor = UIColor.darkGray.cgColor
+            button?.layer.shadowOffset = CGSize(width: 0, height: 2)
+            button?.layer.shadowOpacity = 0.4
+            button?.layer.shadowRadius = 3
+        }
+        
+        let isRightButton = Bool.random()
+        
+        if type != .divide {
+            var randomAnswer: Int
+            repeat {
+                randomAnswer = Int.random(in: (answer - Int.random(in: 1...9))...(answer + Int.random(in: 1...9)))
+            } while randomAnswer == answer
+            rightButton.setTitle(isRightButton ? String(Int(randomAnswer)) : String(answer) , for: .normal)
+            leftButton.setTitle(isRightButton ? String(answer) : String(Int(randomAnswer)), for: .normal)
+        } else {
+            var randomAnswerDivide: Double
+            repeat {
+                randomAnswerDivide = Double.random(in: (answerDivide - Double.random(in: 1.0...9.9))...(answerDivide + Double.random(in: 1.0...9.9)))
+            } while randomAnswerDivide == answerDivide
+            rightButton.setTitle(isRightButton ? String((randomAnswerDivide * 10).rounded() / 10) : String(answerDivide) , for: .normal)
```

Comment:
It would be better to create a local constant for this value String((randomAnswerDivide * 10).rounded() / 10) : String(answerDivide) and use it in the setTitle method. 
It's a little difficult to read and understand.